models/comments.model.js

```diff
@@ -0,0 +1,41 @@
+const db = require("../db/connection");
+const { fetchArticleById } = require("./articles.model");
+
+exports.fetchCommentsByArticleId = async (article_id) => {
+  try {
```

Comment:
I'm less familiar with `try / catch` but I don't think it's necessary to have it in the model, if it fails then it will go to the `catch` of the controller function and be passed to `next`